pkg/services/baremetal/host/host.go

```diff
@@ -1511,29 +1431,35 @@ func (s *Service) actionEnsureProvisioned() (ar actionResult) {
 		if err != nil {
 			return actionError{err: fmt.Errorf(errMsgFailedHandlingIncompleteBoot, err)}
 		}
+		// Often: err: 'failed to dial ssh. Error message: dial tcp 94.130.206.211:22: i/o timeout. DialErr: failed to dial ssh'
```

Comment:
@janiskemper I noticed that this line gets reached if the ssh port is not reachable yet. This is not a big problem. But I think this means that the above line to detect "ssh not reachable" don't work reliably.

Reply:
i/o timeout should be a timeout error. Is it not declared as such?

Reply:
all I can say is that the code reaches this lines, and that it creates the output I added as a comment.

Overall I don't think it makes sense to distinguish between connection-refused and a timeout. A different Linux distribution could return different errors.

I think we should leave it like this in this PR. If we want to, we can create a second PR which handles that better.

@janiskemper what do you think?

Reply:
sorry I don't get it.. This error just means that we need to wait a bit more because the boot process didn't finish.. We will obviously get errors if the server is not reachable. Doesn't really matter whether it's i/o timeout or connection refused. 

All we do, if we haven't reached any time limits, which I believe are handled in handleIncompleteBoot, is to requeue and try later. Which is correct in all cases. 

What you suggest regarding connection refused and timeout errors and to not look at them in such detail, is completely independent of this.

Reply:
I removed the comment and the logger.Info().